src/main/webapp/dev/js/common/instructor.js

```diff
@@ -314,9 +314,11 @@ function bindDeleteButtons() {
         const okCallback = function () {
             window.location = $button.attr('href');
         };
+        const okButtonText = 'Yes';
+        const cancelButtonText = 'No, cancel the operation';
 
         showModalConfirmation('Confirm deleting feedback session', messageText, okCallback, null,
-                null, null, BootstrapContextualColors.DANGER);
+                okButtonText, cancelButtonText, BootstrapContextualColors.DANGER);
```

Comment:
Given that the text is this short, I feel you can just place the strings here instead of creating new variables. It also improves readability. Furthermore, a similar [method](https://github.com/TEAMMATES/teammates/blob/0c0419b5c092bfbf3c5e4a8a1f29535d594e7d3e/src/main/webapp/dev/js/main/instructorCourseStudentEdit.js#L27) is doing it the same way (even though the text is quite lengthy).

Same applies to the other change.

Reply:
Alright!